std/process.d

```diff
@@ -2856,7 +2856,7 @@ void kill(Pid pid, int codeOrSignal)
     do { s = tryWait(pid); } while (!s.terminated);
     version (Windows)    assert(s.status == 123);
     else version (Posix) assert(s.status == -SIGKILL);
-    assertThrown!ProcessException(kill(pid));
+    assert(pid.processID == Pid.terminated);
```

Comment:
Please rewrite this to use ``assertThrown`` as the unittest previously used, unless you have a reason to not do so.

Reply:
Nevermind. Not what I thought it did, ``assert`` is fine.